packages/keyValueCache/src/__tests__/InMemoryLRUCache.test.ts

```diff
@@ -34,12 +40,40 @@ describe("InMemoryLRUCache", () => {
   });
 
   it("does not expire when ttl is null", async () => {
-    await cache.set("forever", "yours", { ttl: null });
+    await cache.set("forever", "yours", { noUpdateTTL: true });
     expect(await cache.get("forever")).toEqual("yours");
 
     await sleep(1000);
     expect(await cache.get("forever")).toEqual("yours");
   });
```

Comment:
Actually it looks like `ttl: null` is no longer allowed (but `ttl: 0` is, and the documentation on `ttl` says 0 is forever). We should update this test to `ttl: 0` and update the test description.

Reply:
@trevor-scheer Thanks for catching this. Yeah, since `LRUCache.SetOptions.ttl` doesn't support null, I had to update the test, but admitted didn't do a good job of actually checking what the test was asserting 😞 

I can just change it to `ttl: 0` as suggested since that's what `LRUCache.SetOptions.ttl` supports, or another option would be to override the `ttl` definition to support null:

```
export type InMemoryLRUCacheSetOptions<
  V extends {} = string,
  FC = unknown,
> = Omit<LRUCache.SetOptions<string, V, FC>, "ttl"> & KeyValueCacheSetOptions;
```

And then always default to `ttl: 0` if it's falsy within the `InMemoryLRUCache.set()` function:

```
async set(key: string, value: V, options?: SO) {
  // If TTL is defined, convert it from seconds to milliseconds.
  // Otherwise default it to 0 to indicate "no TTL".
  this.cache.set(key, value, {
    ...options,
    ttl: options?.ttl ? options.ttl * 1000 : 0,
  });
}
```

This way, we're not breaking the current interface and would still be able to support `ttl: null` as `KeyValueCacheSetOptions` does.

Do you have a preference between the 2 approaches?

Reply:
@trevor-scheer I opted to be backwards compatible and reverted back to be able to support a null TTL value. Let me know what you think.